providers/kit/connector.go

```diff
@@ -44,6 +49,10 @@ func NewConnector(opts ...Option) (conn *Connector, outErr error) {
 	return conn, nil
 }
 
+func (c *Connector) getApiURL(arg string) (*urlbuilder.URL, error) {
+	return urlbuilder.New(c.BaseURL, apiVersion, arg)
```

Comment:
since you use constructURL for escpaing this method could instantiate url that way